keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -469,6 +468,48 @@ message ProcessUsage {
 }
 ```
 
+##### ContainerMetrics additions
+For stats that are outside the scope of /stats/summary but are still reported by cAdvisor, we will return these as unstructured metrics in Prometheus format. The Kubelet will then implement collect methods and descriptors to fetch these metrics from the CRI and export them in Prometheus format. This is done via ListPodSandboxMetrics RPC call.
+
+```
+// ListPodSandboxMetrics gets pod sandbox metrics from CRI Runtime
+rpc ListPodSandboxMetrics(ListPodSandboxMetricsRequest) returns (ListPodSandboxMetricsResponse) {}
+
+message ListPodSandboxMetricsRequest {} 
+
+message ListPodSandboxMetricsResponse {
+    repeated PodSandboxMetrics pod_metrics= 1;
+    repeated ContainerMetrics container_metrics = 2;
+}
+
+message PodSandboxMetrics {
+    string pod_sandbox_id = 1;
+    repeated Metric metrics = 2;
+}
+
+message ContainerMetrics {
+    string container_id = 1;
+    repeated Metric metrics = 2;
+}
```

Comment:
based on the structure of other CRI calls, I think I'd expect this more to be 
```
essage ListPodSandboxMetricsResponse {
    repeated PodSandboxMetrics pod_metrics= 1;
}
message PodSandboxMetrics {
    string pod_sandbox_id = 1;
    repeated Metric metrics = 2;
    repeated ContainerMetrics container_metrics = 3;
}
```

Reply:
Restructured